src/api/todos.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export default axios.create({
+  baseURL: "http://localhost:8000/",
```

Comment:
Instead of hardcoding baseURL, fetch it from env file

Reply:
resolved